src/platform/utilities/api/index.js

```diff
@@ -97,6 +99,12 @@ export function apiRequest(resource, optionalSettings = {}, success, error) {
         ? response.json()
         : Promise.resolve(response);
 
+      // Get CSRF Token from API header
+      const csrfToken = response.headers.get('X-CSRF-Token');
+      if (csrfToken) {
```

Comment:
The CSRF token should stay the same for a given session.
I don't know if it really makes a difference, but you could check that here:
`if (csrfToken && csrfToken != csrfTokenStored) {`